docs/howtoguides/enable_livepatch.md

```diff
@@ -22,13 +42,47 @@ Installing canonical-livepatch snap
 Canonical livepatch enabled.
 ```
 
-To check the status of Livepatch once it has been installed use the following
-command:
+## Check Livepatch status after installation
+To check the status of the Livepatch client once it has been installed use the
+following command:
 
 ```console
 $ sudo canonical-livepatch status
 ```
 
-```{seealso}
-For more information, see: https://ubuntu.com/security/livepatch
+### Unsupported kernels
+
+Although you can enable Livepatch on an unsupported kernel, since patches are
+kernel-specific, you will not receive any updates from Livepatch if your kernel
+is not supported. The `canonical-livepatch status` command will warn you if
+your kernel is unsupported (output truncated for brevity):
+
+```console
+...
+server check-in: succeeded
+kernel state: ✗ kernel not supported by Canonical 
+patch state: ✓ no livepatches needed for this kernel yet
+...
```

Comment:
This works, but `pro status` will give livepatch a "warning" status and include a detailed message if you're on an unsupported kernel, so we should probably point that out first

Reply:
It doesn't on mine - I'm on an unsupported kernel but I only get the warning from `canonical-livepatch status`, but I get no warnings with pro status

Reply:
Resolving, this was a "me" problem, it turns out